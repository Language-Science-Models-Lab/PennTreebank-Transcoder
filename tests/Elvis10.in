
(S @(NNP @Elvis) (. @.))

# tests clause rule 10

# in terminal rule: top of stack is Elvis
# in subword rule: top of stack is type(Elvis, _)
# in word rule 1: top of stack is type(Elvis, _)
# in clause rule 10: top of stack is type(type(Elvis, _), s)


