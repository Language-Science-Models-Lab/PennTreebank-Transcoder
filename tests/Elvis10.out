Elvis:type(type(Elvis, _), s)
